pkg/rtc/transport.go

```diff
@@ -706,19 +706,6 @@ func (t *PCTransport) SetPreferTCP(preferTCP bool) {
 }
 
 func (t *PCTransport) AddICECandidate(candidate webrtc.ICECandidateInit) {
-	if !t.params.Config.UseMDNS {
```

Comment:
push this out to the event handler to be consistent and record it in `connectionDetails`.